db/migrate/20190412130309_add_verdict_and_notes_to_potential_payments.rb

```diff
@@ -0,0 +1,6 @@
+class AddVerdictAndNotesToPotentialPayments < ActiveRecord::Migration[5.2]
+  def change
+    add_column :potential_payments, :verdict, :text
```

Comment:
Think this might be better suited to be a `string`? I don't think we'd need the verdict to be greater than 255 characters

Reply:
I think we should be using `text` pretty much always as there is no difference in performance or storage